### PR TITLE
Remove unnecessary reflect uses

### PR DESCRIPTION
**Pull request by FiloSottile**
TEST THIS, I made it from the GitHub interface. It should compile. If it compiles it should run.

### CODE REVIEW

File: cmd/flarectl/flarectl.go [hunk 1]
```diff
@@ -280,7 +279,7 @@ func zoneRecords(c *cli.Context) {
 		case "MX":
 			r.Content = fmt.Sprintf("%d %s", r.Priority, r.Content)
 		case "SRV":
-			dp := reflect.ValueOf(r.Data).Interface().(map[string]interface{})
+			dp := r.Data.(map[string]interface{})
```
Comment (elithrar) [inline cmd/flarectl/flarectl.go]: Why not `dp, ok` here?

Comment (FiloSottile) [inline cmd/flarectl/flarectl.go]: I just maintained the previous semantic, but yes, better error handling than just a panic would be good.

Comment (jamesog) [inline cmd/flarectl/flarectl.go]: I freely admit that this entire thing is a huge hack and poorly-written.

**Comment by jamesog** [comment]
Finally got around to testing this. LGTM. I agree this needs more error handling, we can fix that in another PR.

**Pull request merged**